### DISCUSSION TITLE
How to add dynamic placeholder?

### DISCUSSION THREAD

**Discussion by thatanjan** [davatorium/rofi#1860] [Q&A] [answered]
Is it possible to add a dynamic placeholder from the cli?

In my theme, I have this section that sets the placeholder: 
```
entry {
    font:   inherit;

    placeholder         : "Search";
    placeholder-color   : @fg2;
}
```

But I want to set a dynamic placeholder text by doing something like this:
```bash
rofi -dmenu --placeholder "Search custom script"
```

Rofi version: 1.7.5+wayland1-41-g61f96e4c (makepkg)
WM/DE: Hyprland

Thanks in advance

**Comment by DaveDavenport** [comment, marked as answer]
you can pass theme snippets using `--theme-str 'placeholder { str: "My text"; }' `

**Reply by DaveDavenport** [reply]
you can also look at : 
https://davatorium.github.io/rofi/guides/DynamicThemes/dynamic_themes/
if you run a new enough version.
Given you run a rofi fork, I am unsure if it is already included.

**Reply by thatanjan** [reply]
Thank you so much @DaveDavenport . Using `-theme-str 'entry {placeholder:"Enter wallpaper url...";}'`
